### PR TITLE
Fix deprecation warnings for react >= 15.5.0

### PR DESCRIPTION
**Pull request by y-yagi**
Hi.

In React v15.5.0, `React.PropTypes` and `React.createClass` became deprecate.
So I migrated to `prop-types` package and `create-react-class`  package.

### CODE REVIEW

File: src/Collapse.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { PropTypes, Children }from 'react';
+import React, { Children }from 'react';
```
Comment (afc163) [inline src/Collapse.jsx]:
```diff
- import React, { Children }from 'react';
+ import React, { Children } from 'react';
```

**Comment by afc163** [comment]
Rebase master

File: src/Collapse.jsx (hunk 1 above)
Comment (y-yagi) [inline src/Collapse.jsx]: I fixed.

**Comment by y-yagi** [comment]
Rebased with master.

File: src/Collapse.jsx [hunk 2]
```diff
@@ -11,7 +12,8 @@ function toArray(activeKey) {
   return currentActiveKey;
 }
 
-const Collapse = React.createClass({
+const createReactClass = require('create-react-class');
```
Comment (shepherdwind) [inline src/Collapse.jsx]: Very thanks for you contribution.

But, I think It is better to change the `createReactClass` code to class style component.

Comment (y-yagi) [inline src/Collapse.jsx]: Thank you for your comment.
I changed the `createReactClass` code to class style component.
Please review.

**Pull request merged**